### PR TITLE
Adding nba_draymond

### PR DESCRIPTION
**Pull request by Janebang**
Adding nba_draymond data to fivethirtyeight package

### CODE REVIEW

**Comment by beanumber** [comment]
@Janebang you need to commit the `man` files as well, and also add yourself to `DESCRIPTION` and update the `NEWS.md`. Please see #54 and #56 .

**Comment by Janebang** [comment]
I am having an error saying installing package DESCRIPTION failed for package 'fivethirtyeight'. Any ideas why?

**Comment by Janebang** [comment]
I figured it out!

**Comment by beanumber** [comment]
Travis says:
```
* checking Rd files ... WARNING

prepare_Rd: man/nba_draymond.Rd:24: unexpected section header '\keyword'

prepare_Rd: man/nba_draymond.Rd:25: unexpected END_OF_INPUT '

```
@Janebang can you please double-check your roxygen tags in `R/data_jane.R`?

**Comment by beanumber** [comment]
Travis says:
```
* checking for code/documentation mismatches ... WARNING

Data codoc mismatches from documentation object 'nba_draymond':

Variables in data frame 'nba_draymond'

  Code: draymond player possessions season

  Docs: DRAYMOND player posessions season

```
These entries have to be spelled exactly the same @Janebang .

**Comment by rudeboybert** [comment]
Hey @Janebang, note you don't need to delete and create a new PR when you need to make changes: just commit/push your changes, and this PR #67 will update itself. Just making sure you know this.

File: R/data_jane.R [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#'
+#' @format A data frame with 3009 rows representing DRAYMOND ratings (Defensive Rating Accounting for Yielding Minimal Openness by Nearest Defender) for every player since the 2013-14 season with 4 variables:
+#' \describe{
+#'   \item{season}{The second year of the season; for example, 2018-2019 season would be listed as 2019}
+#'   \item{player}{Name of the player}
+#'   \item{possessions}{Number of possessions a player during the season}
+#'   \item{draymond}{Defensive Rating Accounting for Yielding Minimal Openness by Nearest Defender}
+#'    }
+#' @source NBA
+"nba_draymond"
```
Comment (rudeboybert) [inline R/data_jane.R]: Could you add a newline here? i.e. there should be a blank line 16.

After you've done this, rebuild the package and make sure the `man/nba_draymond.Rd` file is updated.

**Review by rudeboybert — APPROVED** [review]

**Comment by rudeboybert** [comment]
Merged! Congrats, you're now listed as a [contributor](https://github.com/rudeboybert/fivethirtyeight/graphs/contributors) to the package!

**Pull request merged**